pandas/core/internals/blocks.py

```diff
@@ -2330,6 +2340,12 @@ def get_values(self, dtype=None):
             values = values.reshape(1, -1)
         return values
 
+    def to_dense(self):
```

Comment:
There is a `# TODO(DatetimeBlock): remove` comment on DatetimeLikeBlockMixin.  Can this be merged into that method and the comment removed?

Reply:
It could probably be removed regardless of this PR.

IIUC, DatetimeBlock would fall back to Block.to_dense and use `.values.view()`, while DatetimeTZBlock would fall back to ExtensionBlock and use `np.asarray(self.values)`.